examples/ucnf-kiknos/scripts/pyaws/create_cluster.py

```diff
@@ -0,0 +1,116 @@
+import argparse
+import json
+import subprocess
+import sys
+
+from eks import AwsCluster
+from shell import run_out, run_in
+from utils import reduce_subnets, get_current_region
+
+DEFAULT_CIDR_BLOCK = "192.168.0.0/16"
+
+
+def create_cluster(data):
+    run_in("eksctl", "create", "cluster", "-f", "-", **data)
+
+
+def generate_cluster_cfg(name, region, cidr, vpcid, private, public):
+    return {
+        'apiVersion': 'eksctl.io/v1alpha5',
+        'kind': 'ClusterConfig',
+
+        'metadata': {
+            'name': name,
+            'region': region
+        },
+        'vpc': {
+            'cidr': cidr,
+            'id': vpcid,
+            'subnets': {
+                'private': reduce_subnets(private),
+                'public': reduce_subnets(public)
+            }
+        } if private and public and vpcid else {
+            'cidr': cidr,
+            'nat': {'gateway': 'Single'},
+            'clusterEndpoints': {'publicAccess': True, 'privateAccess': True}
+        },
+
+        'nodeGroups': [
+            {
+                'name': 'cnns-member-ng',
```

Comment:
Drawing attention to the naming here for other reviewers.  Its exposing a bit more of our entire cnns topology than needed for this specific use case,  but I think we can live with it.

Reply:
I don't like the naming here either. Changed.